kill_and_cleanup.sh

```diff
@@ -1,5 +1,16 @@
 #!/bin/bash
 
+# Check for processes using ports 8000 and 8001 and kill them if found
+for port in 8000 8001; do
```

Comment:
should use lock file in /tmp/ or /var/ rather than hardcoded ports

Reply:
https://github.com/CoopHive/Apiary/commit/e3a3918fe5feca0fad05baed485086f6cb5c2817